client/my-sites/pages/page/index.js

```diff
@@ -333,7 +333,7 @@ class Page extends Component {
 		const { page, translate } = this.props;
 		return (
 			<PopoverMenuItemClipboard text={ page.URL } onCopy={ this.copyPageLink } icon={ 'link' }>
-				{ translate( 'Copy Link' ) }
+				{ translate( 'Copy link' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.